front/components/assistant/AssistantList.tsx

```diff
@@ -0,0 +1,65 @@
+import { AssistantPreview } from "@dust-tt/sparkle";
+import type {
+  ConversationType,
+  LightAgentConfigurationType,
+} from "@dust-tt/types";
+import { useRouter } from "next/router";
+import React from "react";
+
+interface AssistantListProps {
+  agents: LightAgentConfigurationType[];
+  handleAssistantClick: (
+    agent: LightAgentConfigurationType,
+    conversation?: ConversationType
+  ) => void;
+}
+
+const AssistantList: React.FC<AssistantListProps> = ({
+  agents,
+  handleAssistantClick,
+}) => {
+  const router = useRouter();
+
+  return (
+    <>
+      <div className="grid w-full grid-cols-2 gap-2 px-4 md:grid-cols-3">
+        {agents.map((agent) => {
+          const href = {
```

Comment:
We should leverage the Link component here so all of those links can be prefetch on the SSR.

Reply:
Actually this is required here anyway so we can pass `shallow` to avoid reloading.